### PR TITLE
Adds i18n support

### PR DESCRIPTION
**Pull request by SidhNor**
- Adds i18n support
- Adds Russian localisation
- Adds the possibility to use English only voice in case the localized TextToSpeach is of poor quality.

### CODE REVIEW

**Comment by SidhNor** [comment]
@voroshkov I would seek some guidance with regards to Russian FPV racing wording as I am used only to English terms :)

**Comment by voroshkov** [comment]
Great PR, dude! Thanks a lot!

File: Android/ChorusRFLaptimer/app/src/main/res/layout/race_result_list_group.xml [hunk 1]
```diff
@@ -64,18 +66,18 @@
...
                     <TextView
                         android:id="@+id/textTotalRaceTime"
                         android:layout_width="0dp"
                         android:layout_height="wrap_content"
-                        android:text="Text"
+                        android:layout_weight="1"
                         android:gravity="center_vertical"
                         android:textSize="12sp"
-                        android:layout_weight="1" />
+                        tools:text="2:12" />
```
Comment (voroshkov) [inline Android/ChorusRFLaptimer/app/src/main/res/layout/race_result_list_group.xml]: can you please clarify what's this? :)

File: Android/ChorusRFLaptimer/app/src/main/res/layout/channel_group.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
             <TextView
                 android:id="@+id/txtDeviceLabel"
-                android:layout_width="70dp"
+                android:layout_width="100dp"
```
Comment (voroshkov) [inline Android/ChorusRFLaptimer/app/src/main/res/layout/channel_group.xml]: why 100? is it better?

Comment (SidhNor) [reply Android/ChorusRFLaptimer/app/src/main/res/layout/channel_group.xml]: Russian text no longer fits and is being cut badly. I suspect other translations wont fit either.

File: Android/ChorusRFLaptimer/app/src/main/res/layout/race_result_list_group.xml (hunk 1 above)
Comment (SidhNor) [reply Android/ChorusRFLaptimer/app/src/main/res/layout/race_result_list_group.xml]: tools:text is the TextView text that is gonna be applied in IDE's "Designer mode" without setting the text in runtime. That way you can play around in design mode with different values, 2:12 means like 2 minutes and 12 seconds

File: Android/ChorusRFLaptimer/app/src/main/res/layout/channel_group.xml (hunk 2 above)
Comment (voroshkov) [reply Android/ChorusRFLaptimer/app/src/main/res/layout/channel_group.xml]: got it!  thanks

**Comment by SidhNor** [comment]
My hands are itching to rewrite some UI as well, but i don't wanna play around with anything before Android 5 (api 21)

**Pull request merged**